packages/amplication-client/src/App.tsx

```diff
@@ -97,6 +103,18 @@ function App() {
     ignoreTags: [],
   });
 
+  const closeConfirmation = useCallback(() => {
+    setWorkspaceUpgradeConfirmation(false);
+  }, []);
+
+  const { trackEvent } = useTracking();
+
+  const trackPurchaseCompleted = useCallback(() => {
+    trackEvent({
+      eventName: AnalyticsEventNames.WorkspacePlanUpgradeCompleted,
+    });
```

Comment:
I expect to see this event in the server side and not in the client side

Reply:
It's going to be changed.